pkg/machines/actions/store-actions.js

```diff
@@ -23,7 +23,6 @@ import {
     SET_HYPERVISOR_MAX_VCPU,
     SET_NODE_MAX_MEMORY,
     SET_LOGGED_IN_USER,
-    SET_PROVIDER,
```

Comment:
This commit should then bump the `Recommends: libvirt-dbus` to `Depends:` in tools/debian/control. Also, I'm not sure if `libvirt-clients | libvirt-bin` is still actually necessary, but I suppose it is -- otherwise we could have done "libvirt-dbus | libvirt-bin" long ago. So let's keep it for now.